posts/2017-07-31-replacing-loops.md

```diff
@@ -273,12 +273,11 @@ Because is an infinite sequence we are taking the first twenty memebers of the s
 
 ## Summary
 
-Writing custom loops is a lot of effort and more often than not there is an alternative already written that can help you out. Sometimes needs to be combined with other functions and sometimes requires small helper functions.
+Writing custom loops is a lot of effort, and more often than not, there is an alternative already written to help you out. Functions often need to be combined with other functions and frequently require small helper functions.
 
-Though _list comprehensions_ and many of the functions we just reviewed are very common in functional languages they also exists in many others. 
+Though the list comprehensions, and many of the functions we just reviewed are very common in functional languages, they also exists in many others language.
```

Comment:
The concept is called _list comprehensions_ ... not sure if `the` should go here...

For example if the word was `recursion` would we say `the recursion` ?

Reply:
you're right. Good catch.